nipoppy/trackers/run_tracker.py

```diff
@@ -3,6 +3,17 @@
 import argparse
 from nipoppy.trackers.tracker import tracker, get_start_time
```

Comment:
The `tracker` class got renamed to `Tracker` to follow Python conventions

Reply:
Yup - just saw the updated class. Fixed the import paths as well.